STIPS/STIP-009.md

```diff
@@ -0,0 +1,101 @@
+
+## Open Questions
+Pose any open questions you may still have about potential solutions here. We want to be sure that they have been resolved before moving ahead with talk about the implementation. This section should be living and breathing through out this process.
+- [ ] When accruing streaming fees where do we send the fees to? The manager?
+    - *Answer*
+- [ ] How do we validate that trades for *x* asset are actually for *x* asset when the trade is encoded in a bunch of bytedata?
+    - *Answer*
+## Feasibility Analysis
+**Single-use vs. Mutli-use Manager Contracts**
+
+Single-use manager contracts would be deployed once per Set Token, while a multi-use manager contract would be deployed once overall and could subsequently be used by all Set Tokens. Single-use manager contracts maintain separation between Set Tokens at the contract level but require individual deployments for each Set Token. A multi-use manager contract requires only one deployment but has functionality across many Set Tokens, which may open up attack vectors.
+
+**Modular vs. Monolithic Manager Contracts**
+
+With modular manager contracts, operators call extension contracts which invoke the manager with arbitrary logic. With monolithic manager contracts, operators call the manager contract directly with logic defined on deployment. The modular manager contracts maintain more flexiblity but require extensions to be deployed and enabled. The monolithic manager contract requires migration for upgradeability but does not need extensions for functionality.
+
+**Individual vs. Global Extensions**
+
+With individual extensions, a new extension contract must be deployed and enabled to provide some functionality to the manager. With global extensions, a collection of extension contracts would be deployed by Set Protocol and made available to managers by simply enabling them. Individual extensions offer more flexibility to managers but require individual contract deployments from managers. Global extensions offer less flexibility but do not require managers to make contract deployments.
+
+**Recommended Solution**
+
+The recommended solution deploys single-use, modular manager contracts with a collection of multi-use, global extensions providing basic functionality. The single-use manager contracts maintain the separation of Set Tokens from each other at the contract level. The modularity of the manager contracts allows for functionaity to be flexible and extensible. The collection of multi-use extensions gives managers access to basic functionality with only a state change and no contract deployment. Manager's will still have the option of deploying individual extensions for more complicated functionality.
```

Comment:
Let's add a quick note on how we'll use the factory to deploy the contracts and also take note of where we'll store the `operator` roles and asset whitelist.

I think that would blend it in a bit better whether the purpose out in the PRD as well as with the background info I wrote above.